<launch>
 <node name="ovpc_mesh_analysis" pkg="mesh_drive_proc" type="mesh_drive_proc_node" output="screen">
     <!--<remap from="/input" to="/voxblox_node/pcl_mesh" />-->
    <remap from="/input" to="/rrt_planner/pcl_mesh" />
    <param name="path_ply" value="$(find mesh_drive_proc)/output/$(anon voxblox_half_edge_).ply"/>
    <param name="max_surface_angle_degree" value="30.0" />
    <param name="local_neighborhood" value="false" />
  </node>

</launch>
